Q1-2026 community board blog (#2547)
writethedocs/www: docs/blog/2026-Q1-community-board.rst

> .. post:: Nov 21, 2025

Author: ```suggestion
.. post:: Mar 2, 2026
```

   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q1-2026
=======================================================

Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share our first quarterly update for 2026.

To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

Q1-2026 meeting highlights
--------------------------

Since this is our first meeting of the year (marking the beginning of our third year of operation as a board, happy boardaversary!), we prepared for the meeting by checking in with the various team leads to find out how the board operations might be improved so that we can better support the teams and the community as a whole.

> **Quarterly meeting format and focus**

Reviewer: I'm not saying we should take this out, but I'm wondering if anyone outside of our group will find this interesting? If we want to include it for transparency, maybe we can put it at the bottom instead?

Since the community board members cover many, MANY time zones, scheduling a time slot for our quarterly meeting has always been a challenge. Our team leads have been exemplary in their dedication, often sacrificing late-nights or early-morning hours to participate.

Over time, we realized that trying to cover all of the time zones had degraded people's energy levels, and we wanted to find a way to to optimize the time that our team leads spend during those meetings for productive or targeted discussions about initiatives and issues, rather than simple roundtable updates (which, let's face it, could be an email). 

As our first experiment of the year, we split the board meeting from a single 1hr meeting that's either late-night or early morning for about half of the board, into 2x30min meetings on the same day for each time zone grouping. Team leads can now choose which meeting they prefer to attend on that day, and thanks to AI note-keeping we are able to capture and share information with each other on Slack/email after the meetings.

> The other goal of having smaller meetings is to facilitate more focused and detailed discussions, and we've already seen the benefit of this approach at our first round of meetings. The team leads were able to bring more detailed topics to each meeting, and even though each meeting was 30min, we were still able to dive more deeply into the topics.

Author: ```suggestion
```

**New initiative: Community content spotlight**

Over the years, we've seen lots of wonderful content published by our documentarians, and occasionally we'd be approached by community members who wish to showcase their content. A while ago we launched the #community-showcase Slack channel, which is well-attended but only reaches WTD Slack members.

On the other hand, the well-loved newsletter, led by Aaron Collier and a wonderful team of volunteers, has been our way to collect pertinent topics from our Slack space and share them on our mailing lists. The newsletter has grown to have some sponsored content and job postings, but still focused only on Slack-generated content.

> As our second experiment of the year, we're investigating a mechanism to periodically collect and publish a spotlight of external content that our documentarians have published on their personal sites or other publications. The goal is to curate WTD community content from outside of our Slack spaces and share them with our mailing lists in some way.

Author: ```suggestion
As our global experiment for the year, we're investigating a mechanism to periodically collect and publish a spotlight of external content that our documentarians have published on their personal sites or other publications. The goal is to curate WTD community content from outside of our Slack spaces and share them with our mailing lists in some way.
```

Since the Australia conference is on hiatus, Swapnil Ogale, the conference chair, has been pursuing possible expansions of the community in APAC in order to build on the foundation in Australia and India. There are many wonderful documentation communities in nearby countries such as Singapore, Japan, and Korea, and Swapnil will be investing time this year in building new relationships with those communities.

If you are a member of or have connections to documentation-related communities in APAC and would like to grow a WTD presence in those regions, please reach out to Swapnil on Slack. 

**Newsletter team evolution**

The newsletter team has been going through some personnel changes during the last year. Some members have departed, while new members have been onboarding. Aaron raised the topic of the next phase of the team and we discussed possible solutions for new long-term volunteers. 

We are also looking into possible ways to expand and diversify the newsletter content, including meetup write-ups and other topics of interest from the community. We hope that the newsletter will continue to serve our community in ways that will also support the team's sustainability.

> 

Author: ```suggestion
**Quarterly meeting format and focus**

Since the community board members cover many, MANY time zones, scheduling a time slot for our quarterly meeting has always been a challenge. Our team leads have been exemplary in their dedication, often sacrificing late-nights or early-morning hours to participate. 

However, this proved to be challenging for focused and targeted discussions, so we decided to split the board meeting from a single 1hr meeting that's either late-night or early morning for about half of the board, into 2x30min meetings on the same day for each time zone grouping. 

Team leads can now choose which meeting they prefer to attend on that day, and thanks to AI note-keeping we are able to capture and share information with each other on Slack/email after the meetings.
```

As always, the board continues to review the community backlog to determine the most valuable efforts for the community in the short and long term. 

What is the community board
---------------------------

The `community board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community teams.

The board members are team leads for each WTD team, as well as the conference chairs of our various conferences:

* Mikey Ariel (CoreOps, brand and design, board chair)
* Eric Holscher (CoreOps, Portland chair)
* Samuel Wright (CoreOps, infrastructure)
* Sasha Romijn (CoreOps, support, Atlantic chair)
* Rose Williams (meetups)
* Janine Chan (Slack moderation)
* Aaron Collier (newsletter)
* Swapnil Ogale (Australia chair)

> The community board meets quarterly in real time, and provides monthly asynchronous status updates. Board members can share the status of what they're working on and request any help to keep the various teams functioning effectively.

Reviewer: ```suggestion
The community board meets quarterly to share what they're working on and request any help to keep the various teams functioning effectively.
```
